docs/docs/guides/local_env/run_more_than_one_pxe_sandbox.md

```diff
@@ -3,6 +3,14 @@ title: Running Multiple PXEs in the Sandbox
 sidebar_position: 2
 ---
 
+:::warning
+
+Unfortunately, this is currently not working due to a bug when attempting to run an `aztec` command with the sandbox running. This will be fixed in a future version.
```

Comment:
Is there a git issue to come back to this line and update (tagging devrel)? Ideally linked to corresponding engineering issue if it exists, so we get notified once fixed.